pkg/gen/ami/ami.go

```diff
@@ -0,0 +1,206 @@
+package ami
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/giantswarm/microerror"
+	"golang.org/x/net/html"
+
+	"github.com/giantswarm/devctl/pkg/gen/input"
+)
+
+var amiTemplate = `// NOTE: This file is generated by devctl. Do not edit.
```

Comment:
I'd like to move it to the bottom of the file.

Reply:
Done.